src/odin/maneuversbuilder.cc

```diff
@@ -2369,6 +2397,76 @@ bool ManeuversBuilder::AreRampManeuversCombinable(std::list<Maneuver>::iterator
   return false;
 }
 
+bool ManeuversBuilder::IsNextManeuverObvious(std::list<Maneuver>& maneuvers,
```

Comment:
I think all the args can be `const` refs? Or are we modifying them somewhere within (I couldn't find any place where we were, but I might've missed something)?

Reply:
made params const